Authentication/SignUp/View/SignupController.swift

```diff
@@ -65,62 +67,120 @@ private extension SignupController {
         bindPasswordElements()
         bindButtons()
         setTextfieldOrder()
+        
+        _viewModel.signUpExecuting.observeNext { [unowned self] executing in
+            if executing {
```

Comment:
I don't like that. I also used to use the conditional operator for this sort of things but now I think that this operator shouldn't be used for control flow. Calling a delegate could potentially imply a side effect. I would prefer to make it more explicit by using `if else`.

I would suggest to only use `?:` when the things that you are returning are values and don't perform side effects. Also they should be short. If the line is too long is better to use `if/else`. Less lines doesn't necessary mean better readability,